saas/urls/provider/metrics.py

```diff
@@ -42,6 +42,8 @@
         name='saas_subscriber_pipeline_download'),
     url(r'^plans/((?P<from_date>\d\d\d\d-\d\d)/)?',
         PlansMetricsView.as_view(), name='saas_metrics_plans'),
+    url(r'^coupons/_download/?$', CouponMetricsDownloadView.as_view(),
```

Comment:
I spoke with @smirolo and we're going to move all the download / export functions into separate view modules. This will push the `download/` component further left, so you'd get URIs like:

1. `/download/coupon/`
1. `/download/coupon/filter_expr`

Sebastien mentioned that this allows us to specify finer-grained permissions for access to the download views.